queries.sql

```diff
@@ -94,3 +94,34 @@ SELECT species, AVG(escape_attempts) AS average_escape_attempts
 FROM animals
 WHERE date_of_birth BETWEEN '1990-01-01' AND '2000-12-31'
 GROUP BY species;
+
+-- Write queries using JOIN
```

Comment:
- [ ] Lastly, in the `queries.sql` file, you were asked to write some queries using `JOIN` and attach screenshots of the results of the queries in your pull request summary. But I noticed that you're currently missing both the queries and the screenshots of the result of the queries so kindly implement this as well and add the requested screenshots in your PR summary :wink: 
- Project Requirement
![image](https://github.com/Roland-Ntwali/Vet-clinic/assets/102699958/93510c97-2f05-4a42-ae01-8eef5fd94027)